.github/workflows/CI.yml

```diff
@@ -0,0 +1,31 @@
+name: Java CI
+
+on:
+  [push, pull_request]
+
+jobs:
+  build-and-test:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 11
```

Comment:
This Java version looks very old. Can you change it to at least 17?

Reply:
Yes! Will have to install 17 on my production server but I'll do that now and update the workflow when it's installed

Reply:
installed & updated